tools/scylla-nodetool.cc

```diff
@@ -3942,15 +3952,17 @@ For more information, see: {}"
         {
             {
                 "repair",
-                "Synchronize data between nodes in the background",
+                "Synchronize vnodes-based data between nodes in the background",
```

Comment:
The term "vnodes-based data" doesn't make sense. I would leave the previous version of this sentence.